fennec-dlc-catalog/run

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash +x
+
+virtualenv venv -p python3
+. venv/bin/activate
+pip install -r requirements.txt
+py.test --env stage cdn-test/
+
+
```

Comment:
once this is running, Jenkins will need output of either a 0 or non-zero to indicate pass or fail.